bigbluebutton-html5/imports/ui/components/video-provider/many-users-notify/container.jsx

```diff
@@ -15,12 +15,14 @@ export default withTracker(() => ({
     hasStream: true,
     role: ROLE_VIEWER,
     presenter: false,
-  }).count(),
-  currentUserIsModerator: Users.findOne({ userId: Auth.userID }).role === ROLE_MODERATOR,
-  lockSettings: Meetings.findOne({ meetingId: Auth.meetingID }).lockSettingsProps,
+  }, { fields: {} }).count(),
+  currentUserIsModerator: Users.findOne({ userId: Auth.userID },
+    { fields: { role: 1 } }).role === ROLE_MODERATOR,
+  lockSettings: Meetings.findOne({ meetingId: Auth.meetingID },
```

Comment:
There's two separate calls to find the Meeting. They should be combined and both properties fetched at once.